### DISCUSSION TITLE
BorderlessFullscreen with transparency

### DISCUSSION THREAD

**Discussion by buekera** [bevyengine/bevy#6933] [Q&A] [answered]
Hey there,

I was wondering if anyone could tell my why setting the `WindowMode::BorderlessFullscreen` option within my `WindowDescriptor` results in a borderless fullscreen with a gray background.
I also add `ClearColor(Color::NONE)` as resource as shown in the example here: https://github.com/bevyengine/bevy/blob/main/examples/window/transparent_window.rs

When not using borderless fullscreen it is working as intended.

Some System Info:
* Manjaro Linux
* Kernel 5.15.81
* x11 Window Manager

Code for window creation:
```
App::new()
        .add_plugins(DefaultPlugins.set(WindowPlugin {
            window: WindowDescriptor {
                mode: WindowMode::BorderlessFullscreen,
                monitor: MonitorSelection::Primary,
                present_mode: PresentMode::AutoNoVsync,
                cursor_grab_mode: bevy::window::CursorGrabMode::None,
                cursor_visible: false,
                transparent: true,
                decorations: false,
                ..default()
            },
            ..default()
        }))
        .run();
```

I also spawn a default camera on startup:
```
commands.spawn(Camera2dBundle::default());
```

Hope someone can shed some light here.

Thanks!

**Comment by tim-blackbird** [comment, marked as answer]
Looks like some settings are only applied when the window is created in windowed mode.
As a work-around, try creating the window in windowed mode and then setting it to borderless afterwards.

I'll make a PR that fixes this in a bit :)
edit: #6934

**Reply by buekera** [reply]
You are 100% correct!

I just added a `startup_system`  like this (maybe not the cleanest):
```
fn setup(
    mut windows: ResMut<Windows>
) {
    let window = windows.primary_mut();
    window.set_mode(WindowMode::BorderlessFullscreen);
}
```

And the BorderlessFullscreen is applied with transparancy.

Thanks for the help and for the PR!
